redis/src/main/scala/com/avsystem/commons/redis/monitoring/ClusterStateObserver.scala

```diff
@@ -0,0 +1,13 @@
+package com.avsystem.commons
+package redis.monitoring
+
+/**
+  * Intended for monitoring client's connections.
+  * Should be non-blocking and handle internal exceptions by itself.
```

Comment:
What about the other observers? I'm missing documentation of the public traits there.

Reply:
Added.